.github/workflows/build-test-application.yml

```diff
@@ -88,3 +88,4 @@ jobs:
       SESSION_SECRET: secret
       SESSION_EXPIRY: 3600000
       SESSION_STORE_TABLE_NAME: account-mgmt-frontend-SessionStore
+      METHOD_MANAGEMENT_BASE_URL: https://method-management-v1-stub.home.dev.account.gov.uk/v1
```

Comment:
Does this actually get called at any point during the tests? If so, we should use build not dev as it's much less likely to be broken